### PR TITLE
first changes

### PR DESCRIPTION
**Pull request by ThorbenKuck**

### CODE REVIEW

File: NetCom2.iml [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <orderEntry type="library" scope="TEST" name="Maven: org.hamcrest:hamcrest-all:1.3" level="project" />
     <orderEntry type="library" scope="TEST" name="Maven: org.hamcrest:hamcrest-all:1.3" level="project" />
```
Comment (Minification) [inline NetCom2.iml]: What's all of this?

Comment (ThorbenKuck) [reply NetCom2.iml]: I don't know, why the heck this is posted.. This file is contained within .gitignore...

Comment (Minification) [reply NetCom2.iml]: Must have been added before the .gitignore was there. Can we remove this file still?

Comment (ThorbenKuck) [reply NetCom2.iml]: Sure, we should be able to. TBD before release 1.0

**Pull request merged**